lib/matplotlib/widgets.py

```diff
@@ -295,12 +295,17 @@ def __init__(self, ax, label, valmin, valmax, valinit=0.5, valfmt=None,
         dragging : bool, default: True
             If True the slider can be dragged by the mouse.
 
-        valstep : float, default: None
-            If given, the slider will snap to multiples of *valstep*.
+        valstep : float or arraylike, default: None
+            If a float, the slider will snap to multiples of *valstep*.
+            If an array the slider will snap to the values in the array.
 
         orientation : {'horizontal', 'vertical'}, default: 'horizontal'
             The orientation of the slider.
 
+        initcolor : color or None, default: 'r'
+            The color of the line drawn at the *valinit* position. If None
+            then no line marking the inital position will be drawn.
```

Comment:
```suggestion
            The color of the line at the *valinit* position. If None then
            no line will be drawn.
```
Can be shorted without loss of information.